smart-table-module/js/Table.js

```diff
@@ -176,11 +185,15 @@
 
             /**
              * insert a new column in scope.collection at index or push at the end if no index
+             * call setDefaultSortColumn to set lastCoulmnSort to already sorted column
              * @param columnConfig column configuration used to instantiate the new Column
              * @param index where to insert the column (at the end if not specified)
              */
             this.insertColumn = function (columnConfig, index) {
                 var column = new Column(columnConfig);
+                if (angular.isDefined(column.reverse)) {
```

Comment:
@hannaxbear , providing `reverse:true` in the column configuration would assign a class to the column header to only show respective sorting icon to user.
Which is useful if the column data received from server/DB is already sorted based on a column metric, in our case *Name* column data will have data sorted in ascending order
Have updated README file to help developers